docs/harbor_compatibility_list.md

```diff
@@ -0,0 +1,51 @@
+# Harbor Compatibility List
+
+This document provides compatibility information for all Harbor components.
+
+## Replication Adapter
+
+|     | Registries       | Pull Mode | Push Mode | Introduced in Release | Automated Pipeline Covered |
+|-----|------------------|-----------|-----------|-----------------------|---------------------------|
+| Harbor|  ![Harbor](img/replication_adapters/harbor_logo.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | Y |
+| distribution | ![distribution](img/replication_adapters/distribution.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | Y |
+| docker hub | ![docker hub](img/replication_adapters/docker_hub.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | Y |
+| Huawei SWR | ![Huawei SWR](img/replication_adapters/hw.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.8 | N |
+| GCR | ![GCR](img/replication_adapters/gcr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | Y |
+| ECR | ![ECR](img/replication_adapters/ecr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | Y |
+| ACR | ![ACR](img/replication_adapters/acr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | N |
+| AliCR | ![AliCR](img/replication_adapters/ali-cr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | N |
+| Helm Hub | ![Helm Hub](img/replication_adapters/helm-hub.png)|![Y](img/replication_adapters/right.png)| N/A | V1.9 | N |
+| Artifactory | ![Artifactory](img/replication_adapters/artifactory.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+| Quay | ![Quay](img/replication_adapters/quay.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+| GitLab Registry | ![GitLab Registry](img/replication_adapters/gitlab.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+
+**Notes**: 
+
+* `Pull` mode is replicating artifacts from specified source registries into Harbor. 
+* `Push` mode is replicating artifacts from Harbor to specified target registries.
+
```

Comment:
* `Pull` mode replicates artifacts from the specified source registries into Harbor. 
* `Push` mode replicates artifacts from Harbor to the specified target registries.

Reply:
Maybe also add a line here to explain what "Automated Pipeline Covered" means, unless you think that this is obvious to users.

Reply:
I think it's obvious. Let's keep it as it is. If there is confusion caused in the future, then we can add it.